spec/fixtures/mockrequest.ts

```diff
@@ -85,6 +85,20 @@ export function generateIdToken(projectId: string): string {
   return jwt.sign(claims, mockKey.private_key, options);
 }
 
+/**
+ * Generates a mocked, unsigned Firebase ID token.
+ */
+export function generateUnsignedIdToken(projectId: string): string {
+  return [
+    { alg: 'RS256', typ: 'JWT' },
+    { aud: projectId, sub: mockKey.user_id },
+    'Invalid signature',
+  ]
+    .map((str) => JSON.stringify(str))
```

Comment:
tip: Up to you if you think this is too dense or neat, but you can also say `.map(JSON.stringify)`

Reply:
woo functional